pkg/worker/check_dns.go

```diff
@@ -405,6 +405,7 @@ func (checker *dnsChecker) processDNSResponse(testStatus testStatus, ips []net.I
 
 	domainExpiry, err := checker.getDNSExpiry()
 	if err != nil {
+		fmt.Printf("error while geting domain expiry time of domain %v, err: %v\n", checker.c.Endpoint, err)
```

Comment:
Use the logger instead of fmt.Printf

Reply:
thanks for the suggetion,
code updated.